### ISSUE TITLE
The currentSite in the email templates does not match the site.

### ISSUE THREAD

**Issue by dowadidi** [verbb/formie#808]
**Description**
`currentSite` in the email templates, returns the primary site instead of the site where the form was submitted.
This might also be the cause why translations don't work?

**Steps to reproduce**
1. Create and set form notification email template with `currentSite.handle` in twig.
2. Submit form on a site other than the primary site

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: Yes

**Additional info**
- Plugin version: 1.5.9
- Craft version: 3.6.18
- Multi-site: Yes

**Additional context**
We're also using `Submission::EVENT_AFTER_SAVE` to save som form submission data to csv.

**Comment by engram-design** [comment]
As email notifications are sent in the queue, where there’s no longer a concept of the “active” or current site, as it’s run later in the process. Either via the CLI, or in Craft later, being in a queue system.

But fortunately, we can set the language and current site when the queue job is run. You could also use `submission.getSite()` to get the site that the submission was made on.

To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.5.12",
  "...": "..."
}
```

Then run `composer update`.

**Comment by dowadidi** [comment]
Thanks! I'll test it as soon as possible.

**Comment by engram-design** [comment]
Fixed in [1.5.13](https://github.com/verbb/formie/releases/tag/1.5.13)
